style.css

```diff
@@ -0,0 +1,479 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+.dNav {
+  transition: 300ms ease-in-out;
+  width: 90%;
+  position: absolute;
+  padding: 5%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
```

Comment:
- [ ] [OPTIONAL] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
   flex-direction: column;
   align-items: flex-start;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉